201-eventhubs-create-namespace-and-enable-archive/azuredeploy.JSON

```diff
@@ -0,0 +1,152 @@
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{  
+      "eventHubNamespaceName":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Name of the EventHub namespace"
+         }
+      },
+      "eventHubName":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Name of the Event Hub"
+         }
+      },
+	  "messageRetentionInDays":{
+		"type":"int",
+		"defaultValue": 1,
+		"minValue":"1",
+		"maxValue":"7",
+		"metadata":{
+		   "description":"How long to retain the data in Event Hub"
+		 }
+	  },
+	  "partitionCount":{
+		"type":"int",
+		"defaultValue":2,
+		"minValue":2,
+		"maxValue":32,
+		"metadata":{
+			"description":"Number of partitions chosen"
+		}
+	  },
+	  "archiveEnabled":{
+		"type":"string",
+		"defaultValue":"true",
+		"allowedValues": [
+		"false",
+		"true"],
+		"metadata":{
+			"description":"Enable or disable the Archive for your Event Hub"
+			}
+	  },
+	  "archiveEncodingFormat":{
+		"type":"string",
+		"defaultValue":"Avro",
+		"allowedValues":[
+		"Avro"],
+		"metadata":{
+			"description":"The encoding format Archive serializes the EventData"
+		}
+	  },
+	  
+	  "archiveTime":{
+		"type":"int",
+		"defaultValue":300,
+		"minValue":60,
+		"maxValue":900,
+		"metadata":{
+			 "description":"the time window for the archival"
+		}
+	  },
+	  "archiveSize":{
+		"type":"int",
+		"defaultValue":314572800,
+		"minValue":10485760,
+		"maxValue":524288000,
+		"metadata":{
+			"description":"the size window for archival"
+		}
+	  },
+	  "destinationStorageAccountResourceId":{
+		"type":"string",
+		"metadata":{
+			"description":"Storage Account resource id where you want the blobs be archived"
+		}
+	  },
+	  "blobContainerName":{
```

Comment:
Does the container need to be pre-created too? Please clarify in the metadata description and the README.